openassessment/xblock/openassessmentblock.py

```diff
@@ -1245,8 +1244,7 @@ def publish_assessment_event(self, event_name, assessment, **kwargs):
             "parts": parts_list
         }
 
-        for key in kwargs:
```

Comment:
Pylint complaining about this key iteration, proposed using `.items()` for iteration or `.update()` for merging.